clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala

```diff
@@ -60,25 +61,32 @@ object ClioClient extends LazyLogging {
 
     client
       .instanceMain(args)
-      .fold(
+      .fold[Unit](
```

Comment:
Why is `Unit` required here? Do the arguments to `fold()` not all have the same type?

Reply:
Probably leftover from debugging a type error.